CMakeLists.txt

```diff
@@ -523,13 +523,13 @@ LINK_DIRECTORIES(${BUILD_BINARY_DIR})
 # Also specify where include files are installed.
 IF(WIN32)
   # Install visualizer to bin, since it needs to be co-located with dll's
-  SET(SIMBODY_VISUALIZER_INSTALL_DIR ${CMAKE_INSTALL_FULL_BINDIR})
+  SET(SIMBODY_VISUALIZER_INSTALL_DIR ${CMAKE_INSTALL_BINDIR})
   # Install include files into base include folder since it's a sandbox
   SET(SIMBODY_INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR})
 ELSE()
   # Visualizer is not intended to be a user executable. Proper place is
   # inside the lib directory
-  SET(SIMBODY_VISUALIZER_INSTALL_DIR ${CMAKE_INSTALL_FULL_LIBEXECDIR}/simbody)
+  SET(SIMBODY_VISUALIZER_INSTALL_DIR ${CMAKE_INSTALL_LIBEXECDIR}/simbody)
```

Comment:
> the package is still relocatable but would then require that SIMBODY_HOME is set correctly

That's right -- the intention was that it would always be able to find the visualizer with no environment variable set as long as the package was installed in the directory specified in CMake at the time it was built. If the installation were moved somewhere else, then it would still work but would require `SIMBODY_HOME` to be set.